test/extended/builds/forcepull.go

```diff
@@ -38,6 +38,16 @@ func doTest(bldPrefix, debugStr string, same bool, oc *exutil.CLI) {
 	// corrupt the builder image
 	exutil.CorruptImage(fullImageName, corruptor)
 
+	if bldPrefix == buildPrefixFC || bldPrefix == buildPrefixTC {
+		// expect custom builds to fail by default
+		_, err := exutil.StartBuildAndWait(oc, bldPrefix)
+		o.ExpectWithOffset(1, err).To(o.HaveOccurred())
```

Comment:
we don't need to confirm that custom builds are disabled by default here, i assume/hope we have other tests for that somewhere as part of your PR to disable them?

so just add the role.